packages/flutter_tools/lib/src/bundle.dart

```diff
@@ -116,6 +116,7 @@ Future<void> build({
         fileSystemScheme: fileSystemScheme,
         packagesPath: packagesPath,
       );
+      kernelBinaryFilename = compilerOutput != null ? compilerOutput.outputFilename : null;
```

Comment:
We have sugar for that: compilerOutput?.outputFilename

Reply:
Done